string benchmark_code
string robot_name
---
bool result